cookbook/form/dynamic_form_modification.rst

```diff
@@ -389,19 +389,19 @@ it with :ref:`dic-tags-form-type`.
 
         # app/config/config.yml
         services:
-            acme.form.friend_message:
+            form.friend_message:
```

Comment:
I think the previous discussions settled on using a ``app.`` prefix for service ids in the doc, to be sure to avoid conflicts with core services